rcl/include/rcl/node.h

```diff
@@ -549,6 +549,17 @@ RCL_PUBLIC
 rcl_ret_t
 rcl_get_disable_loaned_message(bool * disable_loaned_message);
 
+
+RCL_PUBLIC
+RCL_WARN_UNUSED
+rcl_ret_t
+rcl_node_type_description_service_init(rcl_node_t * node);
```

Comment:
It seems like these are helper functions for internal functionality - but they are exposed as a public API here. Maybe they should be `static` in `node.c` instead - or put in `node_impl.h` as methods for the `node_impl`